### PR TITLE
chore: eslint config exivity

### PR DESCRIPTION
**Pull request by Stefanwullems**

### CODE REVIEW

**Comment by Stefanwullems** [comment]
I had some version conflicts in glass when I had them as direct dependencies.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "exivity",
+    "exivity/react",
+    "exivity/testing"
+  ],
+
+  "overrides": [{
+    "files": ["*.stories.*"],
+    "rules": {
+      // Storybook cannot parse the stories otherwise
+      "@typescript-eslint/member-delimiter-style": [2, { "multiline": { "delimiter": "semi"} }]
```
Comment (hongaar) [inline .eslintrc.js]: Instead of 2 you can also use `"error"` which is a little easier to read

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    }
+  }],
+
+  "rules": {
+    "react/function-component-definition": 2
```
Comment (hongaar) [inline .eslintrc.js]: See above

**Comment by hongaar** [comment]
> I had some version conflicts in glass when I had them as direct dependencies.

What you can do, is remove the lockfile from this project. I recently learned this is considered best practice for packages: https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514

**Pull request merged**